packages/@uppy/core/src/getFilePlugins.ts

```diff
@@ -0,0 +1,9 @@
+export default function getFilePlugins(fileDescriptor: {
+  plugins?: string[]
+}): string[] {
+  if (fileDescriptor.plugins) {
+    return fileDescriptor.plugins
+  }
+
+  return []
```

Comment:
```suggestion
  return fileDescriptor.plugins || []
```

Reply:
um,... why is this needed. in what condition would you need to return plugins if its falsey?

Reply:
IIUC my suggestion is exactly equivalent to the code you write, but less wordy

Reply:
oh woops, i thought you were saying.

```
export default function getFilePlugins(fileDescriptor: {
  plugins?: string[]
}): string[] {
  if (fileDescriptor.plugins) {
    return fileDescriptor.plugins
  }

  return fileDescriptor.plugins || []
}
```

I mis-read the diff :sweat_smile:. fixed.